api/components/com_joomlaupdate/src/View/Updates/JsonapiView.php

```diff
@@ -98,20 +98,22 @@ public function prepareUpdate(string $targetVersion): string
      * Run the finalize update steps
      *
      * @param string $fromVersion The from version
+     * @param string $updatefileName The name of the update file
      *
      * @return string  The rendered data
      *
      * @since  5.4.0
      */
-    public function finalizeUpdate($fromVersion)
+    public function finalizeUpdate($fromVersion, $updatefileName)
```

Comment:
```suggestion
    public function finalizeUpdate($fromVersion, $updateFileName)
```
If approved, please change the other instances.